keps/sig-cloud-provider/20190729-building-without-in-tree-providers.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Building Kubernetes Without In-Tree Cloud Providers
+authors:
+  - "@janedoe"
+owning-sig: sig-xxx
```

Comment:
update this

Reply:
done, set sig-cloud-provider

Reply:
Given KEP location I believe this should be sig-cloud-provider and sig-cloud-provider should be removed from the participating section

Reply:
done:
- `sig-cloud-provider` is now the owning sig
- `sig-cloud-provider` is not in participating sigs
- `sig-testing` is in participating sigs (by way of kind will use this, and some subset [me?] will be involved helping to test the out of tree providers with this)